docs/user-guide/release-notes/2.9.x.rst

```diff
@@ -23,3 +23,25 @@ New Features
 * The yum distributor now supports the optional parameter
   ``packages_directory`` which can be used for custom destination directory
   for packages during the publish process.
+
+Known Issues
+------------
+
+Several issues were reported against Pulp 2.8 that were not included in the Pulp 2.9.0 release
+as a result of release timing. The list of :fixedbugs:`2.8.6` outlines these bugs, but
+there is one issue in particular that can potentially break RPM repository syncing after
+upgrading: https://pulp.plan.io/issues/2048
+
+This issue is related to resyncing errata from some repositories, and in a pulp-admin sync
+operation looks like this::
+
+    Task Failed
+
+    Could not parse errata `updated` field: expected format '%Y-%m-%d %H:%M:%S'.
+    Fail to update the existing erratum SOME_ERRATUM_ID.
+
+If you aren't using the errata information in the feed repository, you can update the repo
+to skip errata with ``pulp-admin rpm repo update --repo-id <repo-id> --skip=erratum``.
+
+If you do need errata to be synced from a feed repository and encounter this issue, it will
```

Comment:
s/do need/require/
s/and encounter this issue//
s/it will be fixed in Pulp 2.9.1/consider delaying an upgrade until 2.9.1. This will be fixed in Pulp 2.9.1/